tensorflow/compiler/xla/python/pjrt_ifrt/xla_sharding_serdes_test.cc

```diff
@@ -34,7 +34,7 @@ class XlaShardingSerDesTest : public test_util::ShardingTest {};
 
 TEST_P(XlaShardingSerDesTest, HloShardingRoundTrip) {
   auto device_list = GetDevices({0, 1});
-  auto xla_hlo_sharding = xla::HloSharding::Tile(xla::TileAssignment({2, 1}));
+  auto xla_hlo_sharding = xla::HloSharding::Tile(xla::TileAssignment((absl::Span<const int64_t>){2, 1}));
```

Comment:
Nit: Could you please apply a code formatter? The line can't be longer than 80 characters. Thank you!

Reply:
Done.